tools/run_tests/artifacts/build_artifact_python.sh

```diff
@@ -79,12 +79,12 @@ ${SETARCH_CMD} "${PYTHON}" tools/distrib/python/grpcio_tools/setup.py bdist_whee
 if [ "$GRPC_BUILD_MANYLINUX_WHEEL" != "" ]
 then
   for wheel in dist/*.whl; do
-    "${AUDITWHEEL}" show "$wheel" | tee /dev/stderr |  grep -E -w 'manylinux(1|2010)_(x86_64|i686)'
+    "${AUDITWHEEL}" show "$wheel" | tee /dev/stderr |  grep -E -w "$AUDITWHEEL_PLAT"
```

Comment:
Can you explain about how is this env is set? I can only find its usage.
https://github.com/pypa/auditwheel/blob/c6ff475c3325ae393ffb5872b73cb4583d914a06/auditwheel/main_repair.py

Reply:
Here is a definition of this env: https://github.com/pypa/manylinux/blob/99c19386e670f2aecbe4a5d0397a10a04e2d1eeb/docker/Dockerfile-x86_64#L6